core/src/main/java/org/geysermc/geyser/GeyserImpl.java

```diff
@@ -185,8 +185,13 @@ public void initialize() {
 
         logger.info("******************************************");
         logger.info("");
-        logger.info(GeyserLocale.getLocaleStringLog("geyser.core.load", NAME, VERSION));
+        logger.info(GeyserLocale.getLocaleStringLog("geyser.core.load", NAME, BUILD_NUMBER));
```

Comment:
Maybe instead we could do `Loading Geyser build 420 (git-master-58ff00d)`? Not sure if the version matters too much these days.

Reply:
I agree, for end users, it really doesn't matter - a build number is much clearer. Should i modify the `VERSION` field to include the buildnumber + commit instead of version + commit?
That would be the cleanest imo, but it would change the version field in e.g. fabric.mod.json too. Not sure if that's desired

Reply:
I think that modifying `VERSION` is the best solution there yeah. Especially if it's not used anywhere else.

Reply:
The `${version}` variable is also used in the fabric.mod.json, bungee/spigot plugin yml's, and the velocity plugin annotation. Do we want to change it there too, or make a new variable for it?

Reply:
Per rtm's suggestion, the version would now sow up as the following:
- `2.2.0-SNAPSHOT (git-build-number-e542906)` if a dev version; there is no suitable build number in that case
- `2.2.0-b69 (git-master-e542906)` if it's build 69 from the master branch on the geysermc/geyser repo

What do you think of this change? This would keep it neat and clean; and the `b` prefix for builds is already used in floodgate